libs/rand/random.go

```diff
@@ -17,9 +16,8 @@ const (
 // They all utilize math/rand's prng internally.
 //
 // All of the methods here are suitable for concurrent use.
-// This is achieved by using a mutex lock on all of the provided methods.
+// Different randomness each instance
```

Comment:
? not sure what are trying to say with this sentence

Reply:
I'll have to re-phrase it.

Reply:
I have removed the comment.